python/dgl/graphbolt/impl/in_subgraph_sampler.py

```diff
@@ -0,0 +1,74 @@
+"""In-subgraph sampler for GraphBolt."""
+
+from torch.utils.data import functional_datapipe
+
+from ..subgraph_sampler import SubgraphSampler
+from ..utils import unique_and_compact_node_pairs
+from .sampled_subgraph_impl import FusedSampledSubgraphImpl
+
+
+__all__ = ["InSubgraphSampler"]
+
+
+@functional_datapipe("sample_in_subgraph")
+class InSubgraphSampler(SubgraphSampler):
```

Comment:
Should we add the Fused prefix, considering the future introduction of `Non-Fused SampledSubgraph`?

Reply:
I'm not familiar with that. But why not do the renaming job after the future introduction of `Non-Fused SampledSubgraph`?

Reply:
It is not necessary to rename it, the InSubgraphSampler should not care about which SamplingGraph is used.